### ISSUE TITLE
Wrong default location?

### ISSUE THREAD

**Issue by dabosch** [chubin/wttr.in#459]
Hi,
when I point wttr.in to some nonsense place (such as "asldkhfasjklhfds") or to wttr.in/mylocation the text below the chart shows that the place cannot be found and instead I get the weather forecast for Oymyakon (one of the coldest places...)
The weather report, that I see, doesn't look like Oymyakon though. When querying Oymyakon weather manually, I get a completely different, cooler weather forecast.

Also, when using v2 there is no message at all that the requested location could not be found. I had a typo in my town and was confused by the weather forecast.

![Screenshot from 2020-06-04 14-49-27](https://user-images.githubusercontent.com/55180323/83758733-cf24cf00-a672-11ea-9c91-c2c7747165df.png)

**Comment by chubin** [comment]
Yes, it is true; this bug must be fixed; currently it shows the weather for Los-Angeles instead of Oymyakon. I knew this, but I have still not fixed it. Sorry for that and thank you for reporting

**Comment by chubin** [comment]
It is fixed for v1 now, but not yet for v2, v3 and the rest.
HTTP Status code should be 404 in all of these cases though.
